community/examples/eda/eda-all-on-cloud.yaml

```diff
@@ -0,0 +1,236 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: eda-all-on-cloud
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: eda-all-on-cloud
+  region:      ## Set GCP Region Here ##
+  zone:        ## Set GCP Zone Here ##
+  pool_service_level: "EXTREME"  # Options: "STANDARD", "PREMIUM", "EXTREME"
+  rdma_net_range: 192.168.128.0/18 # Specify an unused CIDR range for the RDMA network
+
+deployment_groups:
+
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+#
+#
+# Deployment Group: Setup
+#
+# Sets up VPC network, persistent NFS shares
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+- group: base
+  modules:
+  # Frontend network for GCE, NetApp Volumes and other services
+  - id: eda-net
+    source: modules/network/vpc
+
+  # Backend RDMA network for GCE instances with RDMA capabilities
+  - id: eda-rdma-net
+    source: modules/network/vpc
+    settings:
+      network_name: $(vars.deployment_name)-rdma-net-0
+      mtu: 8896
+      network_profile: https://www.googleapis.com/compute/beta/projects/$(vars.project_id)/global/networkProfiles/$(vars.zone)-vpc-falcon
+      network_routing_mode: REGIONAL
+      enable_cloud_router: false
+      enable_cloud_nat: false
+      enable_internal_traffic: false
+      subnetworks:
+      - subnet_name: $(vars.deployment_name)-rdma-sub-0
+        subnet_region: $(vars.region)
+        subnet_ip: $(vars.rdma_net_range)
+        region: $(vars.region)
+
+  # PSA is required for Google Cloud NetApp Volumes.
+  # Private Service Access (PSA) requires the compute.networkAdmin role which is
+  # included in the Owner role, but not Editor.
+  # https://cloud.google.com/vpc/docs/configure-private-services-access#permissions
+  - id: private_service_access
+    source: community/modules/network/private-service-access
+    use: [eda-net]
+    settings:
+      prefix_length: 24
+      service_name: "netapp.servicenetworking.goog"
+      deletion_policy: "ABANDON"
+
+  # NetApp Storage Pool. All NetApp Volumes will be created in this pool.
+  - id: netapp_pool
+    source: modules/file-system/netapp-storage-pool
+    use: [eda-net, private_service_access]
+    settings:
+      pool_name: "eda-pool"
+      network_id: $(eda-net.network_id)
```

Comment:
network_id is not required as the `use: [eda-net, private_service_access]` directive will take care of filling in the outputs of `eda-net` into this module